src/plugins/intel_cpu/src/nodes/mkldnn_normalize_node.h

```diff
@@ -39,6 +39,8 @@ struct jit_normalize_call_args {
     size_t dst_stride;
     size_t work_amount;
     size_t oc_off;
+    //ptr to array of post op inputs pointers (flat list)
+    const void* post_op_data;
```

Comment:
minor: ideologically this is `const void** post_op_data;`

Reply:
Indeed, changed. Thanks!